.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Check warnings
         run: |
           cmake -S test -B build -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS_INIT='-DLOG_LEVEL=LOG_LEVEL_DEBUG' \
-          -DCMAKE_C_FLAGS='-Wall -Wextra -Werror'
+          -DCMAKE_C_FLAGS='-Wall -Wextra -Werror -DLOG_LEVEL=LOG_LEVEL_DEBUG'
```

Comment:
Is `-DCMAKE_C_FLAGS_INIT='-DLOG_LEVEL=LOG_LEVEL_DEBUG'` still needed? This seems repeated.

Reply:
Yes, that is not needed and is not the correct CMake variable to set for defining the macro. I have removed it.

Reply:
Thanks.